unit_tests/test_version_utils.py

```diff
@@ -757,3 +758,30 @@ def test_get_branched_repo(scylla_version, distro, expected_repo):
     expected_template = "https://s3.amazonaws.com/downloads.scylladb.com/{}"
     actual_repo = get_branched_repo(scylla_version, distro)
     assert actual_repo == expected_template.format(expected_repo)
+
+
+@pytest.mark.parametrize(
+    "version, expected_repo",
+    (
```

Comment:
Lets set the calendar event in 4 years to meet the "enterprise" `2030.1.0` ^_^

Reply:
that should be accounted for now 😄

Reply:
I expect the `enterprise` thing to be removed till that time, so, not a critical limitation...